ion/bitstream_test.go

```diff
@@ -253,7 +253,7 @@ func TestBinaryReadTimestamp(t *testing.T) {
 		0x80, // second: 0
 		0xCA, // 10 precision units
 		0x2C, // 44
-	}, "2000-01-01T00:00:00.000000004Z", Nanosecond,  UTC)
+	}, "2000-01-01T00:00:00.000000004Z", Nanosecond, UTC)
```

Comment:
(An editor that automatically runs `go fmt` before save has made my life a lot more pleasant, may be worth considering. :) I've been quite happy with VS Code's go mode if you're looking for suggestions.)

Reply:
My vscode is set up to automatically run `go fmt` upon save but my goland requires a 4-key-shortcut to run `go fmt` and I don't always remember to do it as you can see here.

Reply:
In the preferences in GoLand go to Tools -> File Watchers.  You can add one that runs `go fmt`, or better yet `goimports` on save.

Reply:
Thanks for the tip. Hopefully this means I'll never have another 'satisfy formatter' commit again.